### PR TITLE
refactor

### PR DESCRIPTION
**Pull request by JuvitaSaini09**
-Added video library mockbee

### CODE REVIEW

File: src/backend/controllers/HistoryController.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+      {},
+      {
+        error,
+      }
+    );
+  }
+};
+
+/**
+ * This handler handles adding videos to user's history.
```
Comment (AshutoshM29) [inline src/backend/controllers/HistoryController.js:39]: You can remove these comments

**Review by AshutoshM29 — APPROVED** [review]
I don't have any other suggestions as of now, Rest looks good to merge!

File: src/backend/controllers/HistoryController.js [hunk 2]
```diff
@@ -0,0 +1,139 @@
+import { Response } from "miragejs";
+import { requiresAuth } from "../utils/authUtils";
+
+/**
```
Comment (Adarsh-Patel-Dev) [inline src/backend/controllers/HistoryController.js:4]: remove the comments

**Review by Adarsh-Patel-Dev — APPROVED** [review]
Good indentation & well structured code

**Pull request merged**